### PR TITLE
FUSETOOLS2-2103 - create Camel Quarkus project

### PR DESCRIPTION
**Pull request by apupier**
- validation of gav partial

It is picking all Camel files in the root workspace and copying them in
the src/main/resources:camel folder. if there are none, the structure is
still created and it is up to the users to create it.

I think will keep for another iteration:
- allow to open in another workspace the created project https://issues.redhat.com/browse/FUSETOOLS2-2113
- a more complete validation https://issues.redhat.com/browse/FUSETOOLS2-2112

/!\ When a *.camel.xml is used, the generated project will fail when
launching it because Camel Quarkus is not using a recent enough version
of Camel which contains the fix for the extension name.

https://github.com/camel-tooling/camel-lsp-client-vscode/assets/1105127/cd53cf52-a922-4df9-83d1-fa93b6ddfa25

### CODE REVIEW

File: docs/content/commands.md [hunk 1]
```diff
@@ -13,3 +13,13 @@
 ## XML DSL
 
 <p align="center"><img src="../images/camelRouteXMLDSL.gif" alt="Camel Route XML DSL command" class="zoom" width="90%"/></p>
+
+# Create a Camel Quarkus project (using Camel JBang export)
```
Comment (djelinek) [inline docs/content/commands.md:17]: I think it is worthy to mention also in list of features here https://github.com/camel-tooling/camel-lsp-client-vscode/blob/main/docs/index.md?plain=1#L30 and link to commands.md

File: src/ui-test/tests/camel_project_command.test.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+		await driver.wait(async function () {
+			input = await InputBox.create();
+			return (await input.isDisplayed());
+		}, 30000);
+		await input.setText('com.demo:test:1.0-SNAPSHOT');
+		await input.confirm();
+
+		const tree = await sideBar.getContent().getSection('create-camel-project-workspace') as DefaultTreeSection;
+		await driver.wait(async () => {
```
[the remark this answers is not in the document]
Comment (apupier) [reply src/ui-test/tests/camel_project_command.test.ts:66]: added

File: docs/content/commands.md (hunk 1 above)
Comment (apupier) [reply docs/content/commands.md:17]: provided

**Review by djelinek — APPROVED** [review]
Please mention this new feature in test coverage spreadsheet

**Pull request merged**